esper/controllers/device/device.py

```diff
@@ -42,6 +42,10 @@ class Meta:
              {'help': 'IMEI number',
               'action': 'store',
               'dest': 'imei'}),
+            (['-se', '--serial'],
```

Comment:
you can remove -se option. Just go with --serial.

Reply:
or just `-s` ?